README.md

```diff
@@ -1,37 +1,72 @@
-# Unified Model Post-processing: um2nc-standalone
+# Unified Model to NetCDF Post-processing: um2nc
 
 ## About
 
-`um2nc-standalone` is an [ACCESS-NRI](https://www.access-nri.org.au/) project to merge multiple versions of unified model conversion tools to a single, canonical project for the ESM1.5 model. 
+`um2nc` is a `Python3` utility to convert [Unified Model data files](https://code.metoffice.gov.uk/doc/um/latest/papers/umdp_F03.pdf) to NetCDF format.
+
+The `um2nc-standalone` project is an [ACCESS-NRI](https://www.access-nri.org.au/) initiative to merge multiple versions of Unified Model NetCDF conversion tool to a single, canonical project. 
 
 ## Installation
 
-TODO
+### Gadi
 
-* `virtualenv` instructions
-* `conda`/`miniconda`/`micromamba?` instructions
+On Gadi, `um2nc` is available within the `vk83` `payu` environment. 
```

Comment:
Are there any other environments the package is/will be available on? If so, we should probably provide a list (saves a lot of exploring).

Reply:
For now, I don't think there are other environments where the package is available.
In the future, I am not sure, but I would lean towards no anyway.